starwars/src/App.js

```diff
@@ -1,38 +1,22 @@
 import React, { Component } from 'react';
 import './App.css';
+import StarWars from './components/StarWars.js'
+import StarWarsJSON from './components/StarWarsJSON.json'
+
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      starwarsChars: []
+      starwarsChars: StarWarsJSON
```

Comment:
It's cool that you saved the response from the server as a JSON file to help understand the code a bit better! If you're curious as to what the given code was actually doing, you can research the fetch API here:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API